org.scummvm.ScummVM.json

```diff
@@ -0,0 +1,50 @@
+{
+    "app-id": "org.scummvm.ScummVM",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        /* ScummVM does not respect XDG base directories */
+        "--persist=.scummvm",
+        /* Allow Documents directory access - for storing ScummVM games etc. */
+        "--filesystem=xdg-documents"
```

Comment:
It stores games in xdg-documents? That sounds bizarre.

Reply:
I have used this also in my [DOSBox](https://github.com/flathub/com.dosbox.DOSBox) Flatpak. Same as with DOSBox: To use games in ScummVM, you need to manually save them somewhere first and then add them to ScummVM. This will not import/copy them to the ScummVM directory, it just adds them so they still have to be saved in a place where ScummVM has read/write access to them. Documents directory is not ideal for this, but do you have any better suggestion? I really don't want to give it full home directory access.